htdocs/compta/facture/card-rec.php

```diff
@@ -1299,14 +1303,10 @@
 		}
 		print '</tr></table>';
 		print '</td><td>';
-		if ($object->type != Facture::TYPE_CREDIT_NOTE) {
```

Comment:
Because `$object` is not an invoice, it is a recurring invoice, and as far as I know, Dolibarr doesn't have recurring credit notes (I don't even know if that could make sense).
The `FactureRec` object doesn't have a `type` attribute, and checking it generates a warning.